### DISCUSSION TITLE
Pull Through Cache registry-mirror for proxmox

### DISCUSSION THREAD

**Discussion by Sparkxxx** [siderolabs/talos#4924] [General]
In the guide "Configuring Pull Through Cache" there are only 2 high level examples for using the registry on qemu and docker.  
How am I supposed to use the pull trough cache registry for a Proxmox cluster, what files to edit? 

Thank you

**Comment by smira** [comment]
There is no difference on setting up registry mirrors when using Talos on any platform.

Make sure your registry mirror(s) are accessible from the Proxmox environment, and patch the machine configuration accordingly: https://www.talos.dev/docs/v0.14/reference/configuration/#registriesconfig

If using `talosctl gen config`, it has flag `--registry-mirror` to automatically generate proper machine configuraiton.

Registry mirror configuration can also be patched into the running Talos instance (Talos 0.15+ doesn't require a reboot).

**Reply by Sparkxxx** [reply]
Thanks, it works now, missed the docs section

**Comment by Sparkxxx** [comment]
...
